config/routes.rb

```diff
@@ -10,11 +10,15 @@
 
   resources :outcomes do
     resources :assessments
-    resources :direct_assessments
+    resources :direct_assessments, shallow: true
     resources :indirect_assessments
     resources :surveys
     resources :participations
     resources :other_assessments
     resources :outcome_alignments
   end
+
+  resources :direct_assessments, only: [] do
```

Comment:
Is this empty array here kind of a lie? Does the `shallow: true` above mean we actually have `:show`, `:edit`, `:update`, and `:destroy` here? Or does this empty array override those routes that `shallow` is providing?

Since we have to add the `direct_assessments` resource anyway (so we can nest results), maybe it'd be less confusing to be explicit about the actions in each case and ditch the `shallow`?